lib/libzfs_core/libzfs_core.c

```diff
@@ -94,6 +95,16 @@
 #if __FreeBSD__
 #define	BIG_PIPE_SIZE (64 * 1024) /* From sys/pipe.h */
 #endif
+#ifdef _WIN32
+#define	BIG_PIPE_SIZE (64 * 1024)
+#endif
+/*
+ * 64k aswell
+ * https://stackoverflow.com/q/33553837
+ * https://docs.microsoft.com/en-au/windows/win32/api/namedpipeapi/
+ * nf-namedpipeapi-transactnamedpipe
+ */
+
 
```

Comment:
Could the comment go above the `ifdef` and some extra line between FreeBSD and Win32?